pkg/appmanager/resourceConfig.go

```diff
@@ -1664,13 +1648,9 @@ func (appMgr *Manager) createRSConfigFromRoute(
 	var urlRewriteRule *Rule
 	if urlRewrite, ok := route.ObjectMeta.Annotations[f5VsURLRewriteAnnotation]; ok {
 		urlRewriteMap := parseAppRootURLRewriteAnnotations(urlRewrite)
-		if len(urlRewriteMap) > 1 {
-			log.Warningf("URL rewrite annotation: %s does not support multiple values for routes, not processing", urlRewrite)
```

Comment:
I don't see this warning moved anywhere, we still want to use it though, right?

Reply:
This moved to validateResources line 253